src/Compilers/CSharp/Portable/CodeGen/EmitArrayInitializer.cs

```diff
@@ -359,7 +377,7 @@ private bool TryEmitReadonlySpanAsBlobWrapper(NamedTypeSymbol spanType, BoundExp
                 return false;
             }
 
-            var ctor = ((MethodSymbol)this._module.Compilation.GetWellKnownTypeMember(WellKnownMember.System_ReadOnlySpan_T__ctor));
+            var ctor = ((MethodSymbol)this._module.Compilation.GetWellKnownTypeMember(WellKnownMember.System_ReadOnlySpan_T__ctor, recordUsage: false));
```

Comment:
Should we be recording usage if we successfully emit?

Reply:
> Should we be recording usage if we successfully emit?

We don't track anything during emit, because the tracking is limited to binding and lowering, after that we return the result.

---
In reply to: [351368585](https://github.com/dotnet/roslyn/pull/40010#discussion_r351368585) [](ancestors = 351368585)